### DISCUSSION TITLE
Unable to resume paused project

### DISCUSSION THREAD

**Discussion by Mister-SOSA** [supabase/supabase#4930] [Questions]
My project was undergoing a bit of a rework, and understandably, my project was paused. However, I am completely unable to access it through the dashboard to resume it. Upon trying to access the project dashboard, I receive the following error:

`Application error: a client-side exception has occurred (see the browser console for more information).`

The console then reads:

`instrument.js:109 TypeError: Cannot read properties of undefined (reading 'subscription_id')`

The project in question is:

`https://app.supabase.io/project/apkhvjohfmclnxceppmh/`

**Comment by dragarcia** [comment]
Hi @Mister-SOSA 👋 

It appears that your project is still paused. You will need to head over to the `All projects` view over at https://app.supabase.io/ to unpause it. Your project should look as such at the moment:
![image](https://user-images.githubusercontent.com/26374889/149097765-4e8af4bb-8296-4b5a-97cd-bbfbfc2a6065.png)

**Comment by jojule** [comment]
My project got paused as well today. “All project” listing is empty.

**Reply by jojule** [reply]
Resolved: Project appeared in the "All projects" listing (did not take any action, I just needed to wait apparently). Resumed the project successfully.
